packages/n4s/src/rules/__tests__/isValueOf.test.ts

```diff
@@ -0,0 +1,36 @@
+import { enforce } from 'enforce';
+import { isValueOf, isNotValueOf } from 'isValueOf';
+
+const objectToTest = {
+  a: 'Alpha',
+  b: 'Bravo',
+  c: 'Charlie',
+};
+
+describe('isValueOf tests', () => {
+  describe('When "Bravo" is the value of key "b"', () => {
+    it('Should return true using enforce', () => {
+      enforce('Bravo').isValueOf(objectToTest);
+    });
```

Comment:
enforce doesn't return true, it just doesn't throw. Let's change the description to something that's more in line with that. Something like:

```suggestion
    it('Should pass when used in enforce', () => {
      enforce('Bravo').isValueOf(objectToTest);
    });
```